src/args/argument_list.go

```diff
@@ -12,6 +12,7 @@ type ArgumentList struct {
 	sdkArgs.DefaultArgumentList
 	ParamsConfigFile string `default:"" help:"The location of the varnish.params configuration file. If omitted will search in typical install locations"`
 	InstanceName     string `default:"" help:"User defined name to identify data from this instance in New Relic"`
+	VarnishName      string `default:"" help:"Optional. Specify the varnishd working directory (see varnishd -n)"`
```

Comment:
I wonder about why `VarnishName` was chosen as the name of this if the description states that it defined the working directory. Why not `VarnishWorkingDirectory` (Or `VarnishWD`, `VarnishWorkdir`, etc.)

Reply:
Actually is not the working directory. as doc says: 
```
-n name
Specify the name for this instance. This name is used to construct the name of the directory in which varnishd keeps temporary files and persistent state. If the specified name begins with a forward slash, it is interpreted as the absolute path to the directory.
```
Updated the description to reflect exactly when is needed